arrocke/gloss-translation
# refactor: role based access control for views and actions

## What has changed



This focuses on authorizing views and actions within views. I was working on making sure that language admins have access to everything they need, but this is a big enough change that I'll leave the rest for a separate PR.

* Introduces a way to authorize each route according to the users role in the platform and individual language.
* Introduces a way to authorize parts of a view like managing/creating a language
* This requires language memberships to be returned in the user session
* Since users may not be authorized to view a language, we need a proper home view that has no authorization restrictions

## Connected Issues



closes #69 

## QA Steps



1. Log in as an admin and verify that you have access to everything
2. Create a language admin and login. Verify you have access to that language, but not others
3. Create a language translator and login, Verify you have access to that language, but not manage it

## Post-Deployment
